packages/beasties/src/css.ts

```
@@ -80,33 +86,36 @@ export function serializeStylesheet(ast, options) {
   return cssStr
 }
 
+type SingleIterator<T> = (item: T) => boolean | void
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)